ch01/playbooks/webservers.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Configure webservers
+  hosts: webservers
+  tasks:
+    - name: Install nginx
+      apt: name=nginx
+    - name: install config file
+      template: src=nginx.conf.j2 dest=/etc/nginx/ngingx.onf
```

Comment:
I'd like to note that the dest filename probably has a typo ↑

Reply:
Thanks, @Lin-Buo-Ren.  I fixed the typo.